### ISSUE TITLE
TurnOffActivity for Devices, not just Activities [FEATURE]

### ISSUE THREAD

**Issue by SSutherland94** [nicoduj/homebridge-harmony#286]
**Is your feature request related to a problem? Please describe.**
So I've been trying to find more and more ways to streamline my Harmony setup with regard to HomeKit/HomeBridge to make it a tad less clunky, and I was finally onto something! Use activities as this plugin was generally intended. So that was going well, until I found out that 2 activities cannot run in tandem. Namely, when I turn on the activity to get ready to play a game, the activity that gives me an actual On/Off for my Air Conditioner turns off, thus turning my air conditioner off, which on these hot summer days, is not fun.

**Describe the solution you'd like**
What I'd be hoping for, if technically possible, is to have commands, or at the very least, PowerToggle commands, to show TurnOffActivity. Of course I understand that if a device is toggled without HomeKit, that the reflected state will not be accurate.

**Describe alternatives you've considered**
I did consider incorporating my Air Conditioner into each and every activity, but not only would that inevitably turn off the Air Conditioner when the activity ends, it would be less than practical during the other three seasons.

If this is possible at all, that would be awesome, and I would be eternally grateful, as this would allow me, and I'm sure others to reach peak efficiency with greater accuracy on their HomeKit setups! Hope this all made sense.

**Comment by nicoduj** [comment]
You have the option in harmony to specifically say that a device should not be turned off for some activities, would it not be what you need ?

**Comment by SSutherland94** [comment]
Not quite. What I'm looking for is more of a visible, and accurate (depending on individual responsibility of course) stateful power toggle for devices, so for example, I can say "Hey Siri, turn off the monitor", or "Hey Siri, turn the AC off" without having to set up a convoluted scene in the Home app, where if you switch up two words, nothing happens, because the power toggles as we know them now, are stateless, unless they are activity toggles from the Harmony app. Like I would like for my AC power toggle to show it's on/off status in the same way activities currently show their on/off status, if that makes sense.

**Comment by UncleJerm76** [comment]
Hi, new member here. I achieved this by combining the Harmony plug-in with the "Homebridge Dummy" plugin. I recently set this up with my Fireplace after having had the same problem as the OP — that is, if I made the Fireplace an Activity in Harmony it would end the current Activity and vise-versa. And I also didn't want yet another scene. 

1. First in the Homebridge Harmony plug-in I added my Fireplace under the **devicesToPublishAsAccessoriesSwitch** section and I added separate entries for "Fireplace Power On" and "Fireplace Power Off". Doing so added those two buttons respectively to my Apple Homekit screen. But I didn't want to hear myself saying something stupid every time like, "Hey Siri, turn Fireplace On, On" -or- "...Fireplace Off, On" — so this is where the Homebridge Dummy plug-in saves the day. 
3. It works by adding a new (fake) button to your Apple Homekit screen. I simply labeled mine "Fireplace". 
4. After which you would open the fake button in Apple Homekit and assign an Automation to it. In my case it was _[Add Automation/An Accessory is Controlled/Fireplace (this fake button)/Turns On/Fireplace On/On]_ and then of course a second automation to the same fake button for when its flipped off. 

Its been working awesome. Now I just flip the "Fireplace" (fake) button On or say, "Hey Siri, turn the fireplace on" and viola. No interruptions.
